keyboards/xbows/knight/readme.md

```diff
@@ -0,0 +1,14 @@
+# KNIGHT
+![KNIGHT](https://ftp.bmp.ovh/imgs/2019/10/5f357cd665dd4b04.png)
+
+KNIGHT, a split87 and pad Mechanical Keyboard.
+X-BOWS Knight, the first ergonomic mechanical keyboard with magnetic connection of detachable numeric keypad on both sides. Revolutionary seamless integrated aluminum alloy body. Original spherical OEM keycaps. 
```

Comment:
Please add an extra space here so that the markdown renders correctly.

```suggestion
X-BOWS Knight, the first ergonomic mechanical keyboard with magnetic connection of detachable numeric keypad on both sides. Revolutionary seamless integrated aluminum alloy body. Original spherical OEM keycaps.  
```